src/object_types/main_menu.lua

```diff
@@ -343,6 +384,66 @@ return {
             this:draw_element(strings.get('CONFIRM_MENU_OK'),
                               cb.x + cb.w / 2, cb.y + 3 * (cb.h / 4),
                               this.subfont, this.confirm_menu_option == 2)
+        elseif this.state == this.STATE_CONTROLS then
+            -- Render Controls menu.
+
+            local control_text = strings.get('OPTIONS_MENU_CONTROLS')
+            this:draw_element(control_text,
+                              cb.x + cb.w / 2, cb.y * (cb.h / 8),
+                              this.subfont)
+
+            local control_text_jump = strings.get('CONTROL_MENU_JUMP') .. tostring(input.get_key('jump'))
+            this:draw_element(control_text_jump,
+                              cb.x + cb.w / 2, cb.y + 1 * (cb.h / 8),
+                              this.subfont)
+
+            local control_text_left = strings.get('CONTROL_MENU_LEFT') .. tostring(input.get_key('left'))
```

Comment:
Maybe we should consider putting the input key through an uppercase filter as the current menu font doesn't have very readable lowercase characters. But then we'd end up with text like 'LEFT: LEFT' which is equally confusing. Down the road maybe a separate font or key icons would be best.

Reply:
We already have a blank key sprite for the interact prompt. Let's use that.